pymc_marketing/mmm/components/saturation.py

```diff
@@ -369,6 +370,39 @@ class HillSaturation(SaturationTransformation):
     }
 
 
+class RootSaturation(SaturationTransformation):
+    """Wrapper around Root saturation function.
+
+    For more information, see :func:`pymc_marketing.mmm.transformers.root_saturation`.
+
+    .. plot::
+        :context: close-figs
+
+        import matplotlib.pyplot as plt
+        import numpy as np
+        from pymc_marketing.mmm import RootSaturation
```

Comment:
This needs to be added to the pymc_marketing/mmm/__init__.py file for this to work

Reply:
Ahh thanks so much @wd60622 for hand holding me though this - really appreciate it! That's been done ✅

Reply:
My pleasure